upgrade/Stage_0_Prerequisites.md

```diff
@@ -355,6 +355,8 @@ The current Postgres opt-in backups need to be re-generated to fix a known issue
 
```

Comment:
On line 354, should the "This requires a new image" be removed?   Since that image will not be "new" in 1.3.1.

Reply:
fixed..